editorial edits on modifications and proxies in intro (#246)
quicwg/ops-drafts: draft-ietf-quic-manageability.md

design with respect to network treatment, and a description of how common
network management practices will be impacted by QUIC.

> Since QUIC's wire image {{?WIRE-IMAGE=RFC8546}} is integrity protected,

Reviewer: ```suggestion
Since QUIC's wire image {{?WIRE-IMAGE=RFC8546}} is integrity-protected,
```

> Since QUIC's wire image {{WIRE-IMAGE}} is integrity protected, in-network

Author: ```suggestion
Since QUIC's wire image {{?WIRE-IMAGE=RFC8546}} is integrity protected, in-network
```

in-network operations that depend on modification of data are not possible
without the cooperation of an endpoint. Network operation practices that alter
data are only possible if performed as a QUIC endpoint, which might be possible

> with the introduction of a proxy, which either autenticates itself as a

Reviewer 2: ```suggestion
with the introduction of a proxy, which either authenticates as an endpoint.
```

It's not necessary to distinguish acting **for** an endpoint and acting **as** an endpoint like this.  As far as the peer is concerned, they are the same thing.

Reviewer: ```suggestion
with the introduction of a proxy which authenticates as an endpoint.
```

> separate instance or acts as a front-facing instance for the endpoint. Proxy
> operations are not in scope for this document.

Reviewer 2: ```suggestion
Proxy operations are not in scope for this document.
```
